content/components/selectpanel.mdx

```diff
@@ -64,58 +169,329 @@ Select panel should be anchored to its trigger.
+  </Dont>
+</DoDontContainer>
+
+The clear selection [icon button](/components/icon-button) is displayed as soon as at least one item is selected and disappears once the selection is cleared.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/381b2224-3ad0-4320-94cf-8284eb3f5157"
+    />
+    <Caption>
+      When one or more items are selected in a multi-select panel, display a clear button in the header to clear the
+      entire selection..
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/af95eaee-976b-48ab-8fb2-b31389318fca"
+    />
+    <Caption>Don't forget to show a clear button in the header when one or more items are selected.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Seperators
+
+Separators play a crucial role in clearly distinguishing between different sections, particularly when the item descriptions extends over multiple lines. By utilizing separators, the overall scanning experience for users is greatly enhanced.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/3608eee1-5b40-439e-b371-09effe874a48"
+    />
+    <Caption>If one or more items has a block description then add seperators.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/15338fee-fa60-493e-a91c-a74f3502330b"
+    />
+    <Caption>If one or more items has a block description then don't forget to add seperators.</Caption>
+  </Dont>
+</DoDontContainer>
+
+For items that are unlikely to have descriptions that span multiple lines, separators can be omitted to create a lighter experience.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/1ceb26a9-75cb-4ef8-9fa6-73412540f584"
+    />
+    <Caption>If all items have inline descriptions or no description at all then omit seperators.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/a0166db7-0a1c-4641-b664-c318308fe561"
+    />
+    <Caption>If all items have inline descriptions or no description then don't add seperators.</Caption>
+  </Dont>
+</DoDontContainer>
+
+If seperators are added it's important to ensure that the first and last item in the list has no seperator to avoid visual noise.
```

Comment:
I dont feel like this last section is necessary, the component will just handle it. Maxime this is the longest interface guideline I have ever seen 😂

Reply:
I've seen that a few times in Figma files and even the original documentation had this mistake so I got desperate here and tried to clarify that. 😅

Sadly in Figma it can be overridden as well with a toggle. 

I know this is the most complex component I've worked on here so far. 😭

Reply:
I agree with @langermank on this one

Reply:
Haha I will chill down! Removed!